app/components/shared/banner_component.html.erb

```diff
@@ -1,3 +1,5 @@
-<div class="banner" style="background-color:<%= banner.background_color %>;">
-  <%= sanitize link, attributes: %w[href style] %>
-</div>
+<% if banner %>
```

Comment:
That's a good idea! However, some CONSUL users/developers might have a harder time figuring out why a view is not rendered :thinking:. I know I introduced the `render?` method myself a couple of times, but been wondering since then whether we should remove it :thinking:.

So, when in doubt, I think our criteria is to write code which is similar to existing code :thinking:.